DISCOVER/02_minimal_measures.md

```diff
@@ -226,7 +226,7 @@ What to ask on your registration form (and what not to ask)
 
 	-   from a traditionally underrepresented and/or marginalized group in the technology and/or open source communities including, but not limited to persons identifying as LGBTQ, indigenous, women, persons of color, and/or persons with disabilities
 
-	-   and be unable to attend without some financial assistance. (language adapted from [The Linux Foundation](https://www.linuxfoundation.org/event/api-strategy-practice-conference-2017/diversity-scholarship/))
+	-   and be unable to attend without some financial assistance. <!--(language adapted from NOT FOUND[The Linux Foundation](https://www.linuxfoundation.org/event/api-strategy-practice-conference-2017/diversity-scholarship/))-->
```

Comment:
This one could stand as it is a citation of a page that was not crawled by wayback machine. I would suggest we keep it or turn it into a proper citation.